.github/workflows/xtest.yml

```diff
@@ -5,11 +5,6 @@ on:
     types: [xtest]
   schedule:
     - cron: "30 6 * * *"
-  workflow_dispatch:
```

Comment:
TODO figure out how to trigger this from platform -- should it be on sdk release? or on PR? etc

Reply:
I think it should be on any pr that is making changes to the sdk in platform repo. Or in the java-sdk and client-web repo on prs. 

https://docs.github.com/en/actions/using-workflows/reusing-workflows

We leverage reusable workflows internally. @el-virt Can provide some guidance around this.

Reply:
updated, should be callable now https://github.com/opentdf/client-web/pull/310